allel/stats/roh.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, division
+
+import numpy as np
+from hmmlearn import hmm
```

Comment:
Move this import inside the function that needs it, so the hmmlearn dependency is optional.